In file included from apps/traincascade/features.h:4:0,
.                from apps/traincascade/cascadeclassifier.h:5,
.                from apps/traincascade/cascadeclassifier.cpp:1:
apps/traincascade/imagestorage.h:27:9: error: ‘FILE’ does not name a type

--- origsrc/OpenCV-2.1.0/apps/traincascade/imagestorage.h	2010-04-05 20:24:37.000000000 -0500
+++ src/OpenCV-2.1.0/apps/traincascade/imagestorage.h	2010-09-28 00:57:29.275106600 -0500
@@ -2,6 +2,7 @@
 #define _OPENCV_IMAGESTORAGE_H_
 
 #include <highgui.h>
+#include <cstdio>
 
 using namespace cv;
 
